ui/StatusQ/src/StatusQ/Popups/StatusMenu.qml

```diff
@@ -36,13 +36,23 @@ Menu {
     property real maxImplicitWidth: 640
     readonly property color defaultIconColor: Theme.palette.primaryColor1
 
+    property int type: StatusAction.Type.Normal
+
     property StatusAssetSettings assetSettings: StatusAssetSettings {
         width: 18
         height: 18
         rotation: 0
         isLetterIdenticon: false
         isImage: false
-        color: root.defaultIconColor
+        color: {
+            if (!root.enabled)
+                return Theme.palette.baseColor1
+            if (root.type === StatusAction.Type.Danger)
+                return Theme.palette.dangerColor1
+            if (root.type === StatusAction.Type.Success)
+                return Theme.palette.successColor1
+            return Theme.palette.primaryColor1
+        }
```

Comment:
maybe update `root.defaultIconColor`? 
It's used in `StatusSearchLocationMenu.qml`.

Reply:
Should I just set it to:
```qml
readonly property color defaultIconColor: aassetSettingssset.color
```
?